cv_next/app/api/cvs/fetchCvs/route.ts

```diff
@@ -0,0 +1,21 @@
+"use server";
+
+import { filterValues } from "@/types/models/filters";
+import { getPaginatedCvs } from "@/server/api/cvs";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(req: NextRequest) {
+  try {
+    const data = await req.json();
+    const { nextPage, filters }: { nextPage?: number; filters: filterValues } =
```

Comment:
Do we have any user validation on these routes? We want either user validation, or for these safe routes at the very least a rate limit (either here or, if possible, on the platform itself).